samples/MvcSample.Web/Controllers/FormatFilterController.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace MvcSample.Web.Controllers
+{
+    public class FormatFilterController : Controller
+    {
+        public Product GetProduct(int id)
+        {
+            return new Product() { SampleInt = id };
+        }
+
+        [Produces("application/custom", "application/json", "text/json")]
```

Comment:
Copy/paste mistake. I was going to show format and produces filter work together. Will remove this format.